### PR TITLE
Add Swedish translation of README

### PR DESCRIPTION
**Pull request by Nazeeefa**
Added Swedish translation för README.md. Hope it helps :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,9 +81,13 @@
       <td>
         <a href="/docs/README.ja.md">日本語</a>
       </td>
+      <td>
+        <a href="/docs/README.se.md">Swedish</a>
```
Comment (mfix22) [inline README.md]: Translate this to Swedish 🙂

File: README.md [hunk 2]
```diff
@@ -81,9 +81,13 @@
       <td>
         <a href="/docs/README.ja.md">日本語</a>
       </td>
+      <td>
+        <a href="/docs/README.se.md">Svenska</a>
```
Comment (mfix22) [inline README.md]: @Nazeeefa is this right? 🙂

Comment (Nazeeefa) [reply README.md]: @mfix22 yes :)

**Comment by mfix22** [comment]
@all-contributors add @Nazeeefa for translation

**Pull request merged**